packages/expand-button/expand-button.scss

```
@@ -43,17 +45,17 @@
     }
 
     html:not([data-mousenavigation]):not([data-touchnavigation]) &:focus {
+        @include _expanded-arrow(3px);
         border: none;
+        color: var(--jkl-expand-button-focus-color);
 
         &::after {
             box-shadow: 0 0 0 $_focus-ring-width var(--jkl-expand-button-focus-color);
         }
     }
 
-    &:hover,
-    html:not([data-mousenavigation]):not([data-touchnavigation]) &:focus {
-        @include _expanded-arrow(3px);
-        color: var(--jkl-expand-button-focus-color);
+    &:hover {
+        color: var(--jkl-expand-button-hover-color);
```

Her mister vi animasjonen av pila ved hover.

```suggestion
    &:hover {
        @include _expanded-arrow(3px);
        color: var(--jkl-expand-button-hover-color);
```

---

Jeg skjønner ikke hvordan dette fungerer. Her står det at det er resolved, endringen er inne men under reviewers står det at "Requested changes must be adressed".